steward/tests/acceptance.rs

```diff
@@ -30,65 +35,396 @@ use steward::config::StewardConfig;
+    let key_dir = TempDir::new("test_key")?;
+    let config_file_path = key_dir.path().join("config.toml");
+    let mut f = File::create(config_file_path.clone())?;
+    let configu = StewardConfig {
+        cellars: StewardConfig::default().cellars,
+        ethereum: StewardConfig::default().ethereum,
+        keys: KeysConfig {
+            keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+        server: StewardConfig::default().server,
+        gravity: StewardConfig::default().gravity,
+        cosmos: StewardConfig::default().cosmos,
+        metrics: StewardConfig::default().metrics,
+    };
+    f.write_all(toml::to_string(&configu).unwrap().as_bytes())?;
     let mut runner = RUNNER.clone();
-    let mut cmd = runner.arg("start").capture_stdout().run();
-    cmd.stdout().expect_line("Hello, world!");
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "eth",
+            "add",
+            "sha",
+        ])
+        .capture_stdout()
+        .run();
+    // Check that command executes without error.
     cmd.wait().unwrap().expect_success();
+
+    let tmp_path = key_dir.path().to_owned();
+
+    // Check that the temp directory actually exists.
+    assert!(tmp_path.exists());
+    f.sync_all()?;
+
+    Ok(())
 }
 
-/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_with_args() {
+fn eth_keys_delete() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
+    let key_file_path = key_dir.path().join("ethkey.pem");
+    let mut f = File::create(key_file_path.clone())?;
+    f.write_all(PRIVATE_KEY.as_bytes())?;
+    let config_file_path = key_dir.path().join("config.toml");
+    let mut f = File::create(config_file_path.clone())?;
+    let configu = StewardConfig {
+        keys: KeysConfig {
+            keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+        ..Default::default()
+    };
+    f.write_all(toml::to_string(&configu).unwrap().as_bytes())?;
+
+    let mut runner = RUNNER.clone();
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "eth",
+            "delete",
+            "ethkey",
+        ])
+        .capture_stdout()
+        .run();
+    //check that command executes without error
+    cmd.wait().unwrap().expect_success();
+
+    let tmp_path = key_dir.path().to_owned();
+
+    // Check that the temp directory actually exists.
+    assert!(tmp_path.exists());
```

Comment:
Similarly here, the condition we're looking for is that the key file that was deleted no longer exists.